lib/PasswordStrength/PasswordStrength.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import taiPasswordStrength from 'tai-password-strength';
+
+class PasswordStrength extends React.Component {
+  static propTypes = {
+    password: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.strengthTester = new taiPasswordStrength.PasswordStrength();
+    this.strengthTester.addCommonPasswords(taiPasswordStrength.commonPasswords);
+    this.strengthTester.addTrigraphMap(taiPasswordStrength.trigraphs);
+    this.output = 'weak'; // set default password hint to weak
+  }
+
+  componentDidUpdate(prevProps) {
+    // in case password updated and provided
+    if (this.props.password && this.props.password !== prevProps.password) {
+      const password = this.props.password;
+      const result = this.strengthTester.check(password);
+
+      switch (true) {
+        case result.strengthCode.indexOf('WEAK') >= 0:
+          this.output = 'weak';
+          break;
+        case result.strengthCode.indexOf('STRONG') >= 0:
+          this.output = 'strong';
+          break;
+        default:
+          this.output = 'reasonable';
+          break;
+      }
+    }
+  }
+
+  render() {
+    if (!this.props.password) {
+      return null;
+    }
+
+    return (
+      <div aria-live="polite">
+        <FormattedMessage id={'stripes-smart-components.password-strength.' + this.output} />
+      </div>
```

Comment:
It's best that the aria-live region is present already before a message is placed there... The early presence of the aria-live region is what will allow the assistive technology to monitor it for changes so that those can be announced. If the whole thing appears at once, it may not be picked up. You could have it render the empty `aria-live` div by default and then conditionally render the formatted message.

Reply:
Then I can remove <code>aria-live="polite"</code> here and and add it into readme.md example. Am I correct?
Also, can you please suggest how do you test accessibility? I used "Screen Reader" and it works properly.

Reply:
Updated